IMC_Python/Storing_Data_In_Python/Storing_Data_in_Python.md

```diff
@@ -0,0 +1,253 @@
+# Storing Data in Python
+
+Python has the ability to produce useful applications for our potential code; we can create desktop applications, build websites, analyze data, and much more - the possibilities are endless. However, in order to do anything of that caliber with this programming language, you must start from the bottom and master the basics. One essential building block of Python is to learn to store pieces of data that will be useful to our program and, more importantly, decide how we will store this data. Lists, sets, and dictionaries are the most common ways to store data. In this blog post, I will show you how you can use them, why you would want to use them, and what features each data structure has to offer. 
+
+The first data structure I will talk about is probably the most commonly used in Python - lists. With this method, you can store variables of the same type in a specific order. The, you can access and modify them with an index. 
+
+## Lists
+
+### Initialization
+
+*list= [element1, element2, …, elementn]*
+
+When you initialize a list, you must give it a name. If you choose to add elements to the list at this time (optional), they must be separated by commas and wrapped in square brackets, [].
+
+Ex: Take a list of integers called **numbers**. Let’s suppose you want it to contain numbers 0-4. This is how we would initialize the list: 
+
+<center>numbers = [0, 1, 2, 3, 4]</center>
+
+If we wanted **numbers** initialized to be empty, its declaration would look like this:
+
+<center>numbers = []</center>
+
+### Accessing an Element
+
+*list[index]*
+
+The most important thing to keep in mind when using a Python list is how indices are numbered. When we count in programming, we always start at 0, not 1. When we keep track of the index of a list, the first index is always numbered ‘0’. This means that if your list has six elements, the last element is stored at index ‘5’. 
+
+With that being said, we can access an element through the use of square brackets, [], and its index. 
+
+Ex: If we want to access the fourth element (index 3) and save it to a variable **x**, we would call:
+
+<center>x = numbers[3]</center>
+
+“x =” assigns the value of x and “numbers[3]” returns the item at the third index of the list. If you are new to coding, you should keep in mind that **“=” does not indicate an equivalence relation, it means that a value is being assigned to the variable on the left.** Looking at the example above, we can say that *x is numbers[3]* or *x is 3*, BUT, we CANNOT say that *numbers[3] = x*. The variable on the right is not modified at all. Therefore in this case, although we change what ‘x’ means, the value of numbers[3] remains the same.
+
+### Modifying an Element
+
+*list[index] = new element*
```

Comment:
I like this format you have of of 
Heading
Code general
Information
Code example and comments

However, the code and comments seem a little mixed with the bold/italicized to reference different info. Is there a way you could indent or highlight code areas? Just to make them more distinguishable.

Reply:
Sorry! When I did it on my computer, I used <center> to center the code but I guess it didn't translate when I uploaded to GitHub. I've fixed it now